FirebaseInAppMessaging/Swift/Tests/Integration/FIAMSwiftUI/Podfile

```diff
@@ -0,0 +1,15 @@
+source 'https://github.com/firebase/SpecsStaging.git'
```

Comment:
An initial first line should be inserted:

`source 'https://github.com/firebase/SpecsDev.git'`

See other Podfiles in the repo for an example

Reply:
Resolved in 5d8b009.